src/users/user.dto.ts

```diff
@@ -1,5 +1,9 @@
 export class UserDto {
-  id: string;
```

Comment:
why did you remove the id?

Reply:
It's probably because you used it for createUser input. You should keep the id here and create a new one: createUserDto with just email and password.